hack/components.sh

```diff
@@ -17,7 +19,8 @@ KUBEMACPOOL_PATH=${GOPATH}/src/${KUBEMACPOOL_REPO}
 
 echo 'Fetch kubemacpool sources'
 (
-    go get ${KUBEMACPOOL_REPO} || true
+    mkdir -p ${KUBEMACPOOL_PATH}
+    git clone ${KUBEMACPOOL_URL} ${KUBEMACPOOL_PATH} || true
```

Comment:
how can this fail ? (other than no internet connectivity ? I mean that possibly the `|| true` is not needed when using w/ git.

sorry for nit-picking, but I think this belongs on a different commit (no need to do it if in the end all commits get squashed into a single one).

Reply:
You are so right. Thanks for fighting my lazy ass! 

https://github.com/kubevirt/cluster-network-addons-operator/pull/366